javamelody-spring-boot-starter/src/main/java/net/bull/javamelody/JavaMelodyAutoConfiguration.java

```diff
@@ -143,15 +142,6 @@ public FilterRegistrationBean monitoringFilter(JavaMelodyConfigurationProperties
 	}
 
 	/**
-	 * @return DefaultAdvisorAutoProxyCreator
-	 */
-	@Bean
-	@ConditionalOnMissingBean(DefaultAdvisorAutoProxyCreator.class)
-	public DefaultAdvisorAutoProxyCreator defaultAdvisorAutoProxyCreator() {
```

Comment:
This is the cause of the proxy taget class not working correctly and break `@Scheduled` annotion